mappings/net/minecraft/server/network/ServerPlayerEntity.mapping

```diff
@@ -7,6 +7,7 @@ CLASS net/minecraft/class_3222 net/minecraft/server/network/ServerPlayerEntity
 	FIELD field_13969 seenCredits Z
 	FIELD field_13970 advancementManager Lnet/minecraft/class_2985;
 	FIELD field_13972 syncedSaturationIsZero Z
+	FIELD field_13973 levitationStartTime I
```

Comment:
`levitationStartAge` would be better since `lastActionTime` refers to the system time but this refers to the age of the entity.

Reply:
age? tick?

Reply:
Something like that but `time` is inaccurate.

Reply:
```suggestion
	FIELD field_13973 levitationStartTick I
```
then. Age is definitely a bad term for ticks.